checked/debug.go

```diff
@@ -189,7 +193,7 @@ func (d *debugger) append(event debuggerEvent, ref int, pc []uintptr) {
 			// Shift all tracebacks back one if at end of traceback cycles
 			slice := d.entries[0]
 			for i, entry := range slice {
-				tracebackCallersPool.Put(entry.pc)
+				tracebackCallersPool.Put(&entry.pc)
```

Comment:
Yea, good point, we need to change the field in `entry` to be a pointer. Will update.